packages/server/lib/middleware/access.middleware.ts

```diff
@@ -392,6 +392,27 @@ export class AccessMiddleware {
 
         next();
     }
+
+    internal(req: Request, res: Response, next: NextFunction) {
```

Comment:
I am not sure about the `internal` name and API endpoint prefix below. Anybody has a better idea?

Reply:
I've always used `internal` or `private`. I think I prefer `internal` though.

Reply:
Looks good to me:
public = public
private = dashboard
internal = only for us